.github/workflows/ci.yml

```diff
@@ -252,6 +259,12 @@ jobs:
           - typo3-version: "^12.4"
             php-version: "8.3"
             composer-dependencies: highest
+#          - typo3-version: "^12.4"
+#            php-version: "8.4"
+#            composer-dependencies: lowest
```

Comment:
Why do we keep this commented out?

Reply:
Because it fails: https://github.com/TYPO3BestPractices/tea/actions/runs/13112021223

Reply:
This is due to an incompatible Symfony DI version, and requiring a higher version leads to problems in other components, which then also would need to be required in higher versions etc. etc., which opens a whole can of worms.

Reply:
Ah okay … yeah fine for me for now, as highest is working already.